packages/react-components/react-swatch-picker-preview/tsconfig.lib.json

```diff
@@ -16,7 +16,9 @@
     "**/*.test.ts",
     "**/*.test.tsx",
     "**/*.stories.ts",
-    "**/*.stories.tsx"
+    "**/*.stories.tsx",
+    "**/*.cy.ts",
+    "**/*.cy.tsx"
   ],
-  "include": ["./src/**/*.ts", "./src/**/*.tsx"]
+  "include": ["./src/**/*.ts", "./src/**/*.tsx", "cypress.config.ts"]
```

Comment:
this is incorrect. have you used generator for this setup ?

Reply:
This one I added manually because tests didn't work. Do you mean generator for the component or there's a separate generator for tests?